core/backend/src/workspace/SettingsSchemas.ts

```diff
@@ -21,208 +18,52 @@ import { IModelJsFs } from "../IModelJsFs";
  * @beta
  */
 export interface SettingSchema extends Readonly<JSONSchema> {
-  readonly items?: SettingSchema; // must be single object, not array
-  readonly type: JSONSchemaTypeName; // type is required for settings
+  /** Entries for an array. Must be single object, not array */
+  readonly items?: SettingSchema;
+  /** Type is required for settings */
+  readonly type: JSONSchemaTypeName;
+  /** Name of typeDef for objects or arrays to inherit */
+  readonly extends?: string;
+  /** For objects, a list of named properties and their definitions */
   readonly properties?: { [name: string]: SettingSchema };
-
-  /** whether the setting replaces lower priority entries with the same name or combines with them. */
-  readonly cumulative?: true;
+  /** Specifies how to treat lower-priority entries with the same name for arrays.
+   * By default, like other types of settings, the setting uses the highest-priority entry.
+   * If `combineArray` is `true`, then the setting combines all of the arrays into one array, ordered by priority and eliminating duplicate values.
+   * Duplicate values are determined by the `===` operator - objects and nested arrays are not compared for deep equality.
+   */
+  readonly combineArray?: boolean;
 }
 
 /**
  * The properties of a group of [[SettingSchema]]s for an application. Groups can be added and removed from [[SettingsSchemas]]
- * and are identified by their (required) `groupName` member
+ * and are identified by their (required) `schemaPrefix` member
  * @beta
  */
 export interface SettingSchemaGroup {
-  readonly groupName: string;
-  readonly properties: { [name: string]: SettingSchema };
+  readonly schemaPrefix: string;
+  readonly settingDefs?: { [name: string]: SettingSchema | undefined };
+  readonly typeDefs?: { [name: string]: SettingSchema | undefined };
   readonly order?: number;
```

Comment:
`order` does not appear in Base.Schema.json, is undocumented here, and appears to be unused anywhere. I infer that it is used for ordering groups in a UI, with lower values being ordered before higher values. Correct?

Reply:
Yes, that's its only purpose.

Reply:
Added to schema and documented.